README.md

```diff
@@ -207,8 +229,23 @@ following attributes:
 
 * `file`: Optional. The generated filename this source map is associated with.
 
+The promise of the constructed souce map consumer is returned.
+
+When the `SourceMapConsumer` will no longer be used anymore, you must call its
+`destroy` method.
+
+```js
+const consumer = await new sourceMap.SourceMapConsumer(rawSourceMapJsonData);
+doStuffWith(consumer);
+consumer.destroy();
+```
+
+#### SourceMapConsumer.prototype.destroy()
+
+Free this source map consumer's associated wasm data that is manually-managed.
+
 ```js
-var consumer = new sourceMap.SourceMapConsumer(rawSourceMapJsonData);
+consumer.destroy();
```

Comment:
Bummer but I guess there's no good way around it.

Reply:
I have ideas to improve this in follow ups:

Either add an async RAII-ish function like:

```js
SourceMapConsumer.with = async function (rawSourceMap, f) {
  const consumer = await new SourceMapConsumer(rawSourceMap);
  try {
    await f(consumer);
  } finally {
    consumer.destroy();
  }
};
```

Or alternatively give every `SourceMapConsumer` its own wasm module instance, make the `Mappings` a global/implicit in the wasm heap, and let the GC manage lifetimes. Requires further investigation.